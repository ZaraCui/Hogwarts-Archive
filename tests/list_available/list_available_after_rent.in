ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Harry Potter
RENT 100000 1001
LIST AVAILABLE
EXIT

