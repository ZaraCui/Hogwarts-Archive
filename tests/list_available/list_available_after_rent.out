user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: Dark Arts Basics (Salazar Slytherin)
Patronus Guide (Godric Gryffindor)
Transfiguration I (Minerva McGonagall)
Advanced Charms (Filius Flitwick)
Herbology Field Notes (Newt Scamander)
Polyjuice Manual (Libatius Borage)
Transfiguration i (Minerva McGonagall)
patronus guide (Godric Gryffindor)
Dark Arts Basics (Salazar Slytherin)
Levitation Charms (Helga Hufflepuff)
Advanced Charms (Filius Flitwick)

user: Ending Archive process.
